src/Orc.Controls/Orc.Controls.Shared/Controls/DateTimePicker/Views/DateTimePickerControl.xaml.cs

```diff
@@ -45,18 +50,36 @@ public DateTimePickerControl()
                 NumericTBMinute,
                 NumericTBSecond,
             };
+        }
 
-            NumericTBDay.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBMonth.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBYear.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBHour.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBMinute.RightBoundReached += NumericTextBoxOnRightBoundReached;
+        private void SubscribeNumericTextBoxes()
+        {
+            _numericTextBoxes[0].RightBoundReached += NumericTextBoxOnRightBoundReached;
```

Comment:
This code looks a bit weird, maybe it's smart to add a comment so we won't "fix this bug" in a rush in the future.